domains/ecoplayer07.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "crazyguy66",
+           "email": "kareemehab910@gmail.com",
+           "discord": "962048694293762108"
+        },
+    
+        "record": {
+            "A": ["41.46.72.138"]
```

Comment:
cant access website

Reply:
its open wdym

Reply:
i cant visit website, are you sure its the correct IP or is website in a port ip?

Reply:
isnt the ip supposed to be ipv4?